ut/alarm_req_listener_test.cpp

```diff
@@ -249,8 +249,18 @@ inline Matcher<netsnmp_variable_list*> TrapVars(TrapVarsMatcher::TrapType trap_t
   return MakeMatcher(new TrapVarsMatcher(trap_type, alarm_index));
 }
 
+TEST_F(AlarmReqListenerTest, ClearAlarmNoSet)
+{
+  EXPECT_CALL(_ms, send_v2trap(TrapVars(TrapVarsMatcher::CLEAR,
+                                        1000)));
+
+  _alarm_1._clear_state.issue();
+  _ms.trap_complete(1, 5);
+}
+
 TEST_F(AlarmReqListenerTest, SetAlarm)
 {
+  advance_time_ms(AlarmFilter::ALARM_FILTER_TIME + 1);
```

Comment:
Is there a test where you set an alarm, wait >5s, then set it again, and check you only get one trap? If not, could you add one?